multiversx_sdk_cli/cli_shared.py

```diff
@@ -75,8 +78,8 @@ def add_tx_args(args: List[str], sub: Any, with_nonce: bool = True, with_receive
     if with_data:
         sub.add_argument("--data", default="", help="the payload, or 'memo' of the transaction (default: %(default)s)")
 
-    sub.add_argument("--chain", default=config.get_chain_id(), help="the chain identifier (default: %(default)s)")
-    sub.add_argument("--version", type=int, default=config.get_tx_version(), help="the transaction version (default: %(default)s)")
+    sub.add_argument("--chain", help="the chain identifier")
+    sub.add_argument("--version", type=int, default=2, help="the transaction version (default: %(default)s)")
```

Comment:
We can define a constant here.

Reply:
created a constant `DEFAULT_TX_VERSION = 2`